doc/book/link_routing.adoc

```diff
@@ -164,3 +164,37 @@ link-routing.
 The above configuration allows Rb and Rp to reject attaches that should
 be routed to B2 with an error message that indicates that there is no
 route available to the destination.
+
+Automatic Link Routes
+~~~~~~~~~~~~~~~~~~~~~
+
+There is an alternative way to use link routes that doesn't involve any
+configuration.  An endpoint connected to a router may declare, using
+connection properties, its availability to accept routed links for a
+specific set of addresses.  This connection may be connected to the router
+through a normal listener.  There is no need to create a listener with the
+'route-container' role.
+
+The connection property that controls this feature is
+'qd.link-route-patterns'.
+
+To use the feature, a client provides a connection property when opening
+the connection to the router.  The property has the key
+'qd.link-route-patterns'.  The value is a list containing an even number of
+elements, arranged in pairs.  The first element in a pair is a string
+containing the link route pattern.  This uses the same syntax that would be
+used in a linkRoutePattern configuration item.  The second element in the
```

Comment:
'linkRoutePattern' config item has been deprecated and removed from schema. Currently 'linkRoute' is used.

Reply:
Good point.  I meant the pattern attribute for linkRoute.  This needs to be clarified.